src/app/qml/KeyboardRows/JsonTranslator.qml

```diff
@@ -48,17 +48,17 @@ QtObject {
         case shortName:
             var translation = "";
             if (id === "ctrl_keys") {
-                // TRANSLATORS: This the short display name of a keyboard layout. It should be no longer than 4 characters!
-                translation = i18n.tr("CTRL");
+                // TRANSLATORS: This the short display name of a keyboard layout.
+                translation = i18n.tr("Ctrl");
             } else if (id === "fn_keys") {
-                // TRANSLATORS: This the short display name of a keyboard layout. It should be no longer than 4 characters!
-                translation = i18n.tr("FNS");
+                // TRANSLATORS: This the short display name of a keyboard layout.
+                translation = i18n.tr("Fn");
             } else if (id === "scroll_keys") {
-                // TRANSLATORS: This the short display name of a keyboard layout. It should be no longer than 4 characters!
-                translation = i18n.tr("SCR");
+                // TRANSLATORS: This the short display name of a keyboard layout.
+                translation = i18n.tr("Scr");
             } else if (id === "simple_cmds") {
-                // TRANSLATORS: This the short display name of a keyboard layout. It should be no longer than 4 characters!
```

Comment:
why did you edited all this TRANSLATORS comment?

Reply:
This one looks a bit odd. Maybe it isn't a hard limit, or erroneously I thought it wasn't necessary?

Reply:
This is still not acceptable. See my comment above, as long as you just change the code nothing will happen. And if you commit the pot file it will break translations for all languages due to the new strings. You really want all people to have to restranslate just because you think it should be lowercase?